files/userSurvey.tex

```diff
@@ -0,0 +1,36 @@
+\chapter{User survey}
+In the initial phase of the project a user survey was distributed over social media to get an understanding of the product user group.
+In the period August 29th 2016 to September 20th 2016, the survey received 529 responses of which 425 was users playing badminton.
+
+\subsection*{Racket price}
+When comparing the price ranges to user groups it shows that most users have a racket in the price range kr. 1000 - 1499 as seen on figure \ref{fig:survey:priceVsAge}.
+It is interesting that in the age of 19 to 24 years users spend the most money on their rackets.
+The price ranges can be used to find a proper cost for the product and/or services offered.
+
+\graphic{1}{userSurvey/priceVsAge}{Price distribution versus age groups}{fig:survey:priceVsAge}
+
+\subsection*{Gadget knowledge}
+When asking what sport-gadgets the users know, the results show a clear pattern as seen on figure \ref{fig:survey:gadget}.
+\Glsi{endomondo} is the most known gadget closely followed by \glsi{gopro} with 87\% and 73\% respectively.
+A key difference between these are clear when looking at how many users owning it.
+Only 17\% of the users knowing \gls{gopro} owns one whereas it is 52\% with \gls{endomondo}.
+An explanation on this could be that \gls{endomondo} has a free version of the application with limited features.
+\Glsi{gopro} offers no such thing as it is a physical product.
+
+\graphic{1}{userSurvey/gadget}{Knowledge and ownership of sport-gadgets}{fig:survey:gadget}
+
+\subsection*{Information sharing}
+The survey asked the users if they were willing to share their recorded information to improve the algorithms behind the calculated results. 
+81\% approved this, figure \ref{fig:survey:share}.
+Further more the users was asked if they would share their recordings to match other players for ranking and alike and only 62\% approved this.
+As the difference between these two options are quite subtle, i.e. the anonymous part of it, it is clear that this must be taken into account when creating the product.
+ 
+\graphic{1}{userSurvey/share}{Connection of anonymous data to improve algorithm and user data to allow ranking}{fig:survey:share}
+
+\subsection*{Implementation options}
+When asking for possible sensor implementation solutions for mounting the sensors to the racket, the most popular option with 57\% of the answers, was mounting a sensor on an existing racket, as seen on figure \ref{fig:survey:implementation}. 
+This is the least intrusive solution as well, why this was expected.
+The initial approach from the consultants was to mount the sensor inside a racket.
+This is approved by 36\% of the users, why this is a viable solution for the prototype.
```

Comment:
make it more clear that 36 % are people who are willing to buy a new racket with the sensor build in